django/Dockerfile

```
@@ -7,16 +7,17 @@ ENV PYTHONDONTWRITEBYTECODE=1 \
 
-# Install python dependencies in /usr/local/
+# Install dependencies
 WORKDIR /cohiva
-# Copy only the necessary files to improve build caching
-COPY requirements-docker.txt requirements.txt
-COPY install_dependencies.sh install.py ./
-COPY geno/python-sepa geno/python-sepa
-COPY cohiva/saml2/*.patch cohiva/saml2/
-RUN chmod +x ./install_dependencies.sh
-RUN ./install_dependencies.sh -e docker
+COPY pyproject.toml uv.lock /cohiva
```

Hadolint (2.15.1)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)